manifests/account_config.pp

```diff
@@ -25,7 +26,8 @@
     group   => 'root',
     mode    => '0644',
     content => epp('helm/tiller-serviceaccount.yaml.epp', {
-      'service_account' => $service_account,
+      'service_account'    => $service_account,
+      'image_pull_secrets' => $tiller_image_pull_secrets,
```

Comment:
add a check to only add this line unless `$tiller_image_pull_secrets == undef` ?

Reply:
If the configuration accepts an empty array, setting the default in params.pp to `[]` would probably also be an option.

Reply:
You're fully right, sorry for it, will do